test/verify/check-machines

```diff
@@ -392,7 +392,15 @@ class TestMachines(MachineCase, StorageHelpers, NetworkHelpers):
         b.wait_in_text(".memory-usage-chart .pf-c-progress__status > .pf-c-progress__measure", "0 /")
         b.wait_in_text(".vcpu-usage-chart .pf-c-progress__status > .pf-c-progress__measure", "0%" )
 
-        self.goToMainPage()
+        # shut off of a transient VM will redirect us to the main page
+        m.execute("virsh dumpxml subVmTest1 > /tmp/subVmTest1.xml")
+        m.execute("virsh start {0} && virsh undefine {0}".format("subVmTest1"))
+        b.reload()
```

Comment:
Why does it need a reload here? Isn't that kind of cheating? If it's because of some limitation of libvirt-dbus (no signal on transient shutoff?), please add a comment why it's necessary. Or maybe you meant `wait_page_load()`?

Reply:
We do get an event when a VM get's undefined while running, thus bacoming transient. And we properly update the UI.

But currently we don't have any indicator on the UI or the DOM tree which I can wait for till I am sure that the internal VM state object already updated according to the event, and we are ready to proceed with clicking the shut off button.

I can also add some info on the UI about transient VMs - that would benefit the user and we could rely on that property to wait until we proceed.

Reply:
But in this case, wouldn't a reload() still be a race condition? Could it happen that the reload finishes before the transient VM stops? So maybe adding at least some internal DOM data-* attribute would be good here.

Reply:
Added the data-row-transient

Reply:
Thanks, this looks great and more robust! I'm happy to :+1 after you hit "ready for review".